pkg/util/liveconfig/hive_test.go

```diff
@@ -6,69 +6,124 @@ package liveconfig
 import (
 	"context"
 	"embed"
+	"errors"
+	"net/http"
 	"testing"
 
-	mgmtcontainerservice "github.com/Azure/azure-sdk-for-go/services/containerservice/mgmt/2021-10-01/containerservice"
-	"github.com/Azure/go-autorest/autorest/to"
-	"go.uber.org/mock/gomock"
+	azfake "github.com/Azure/azure-sdk-for-go/sdk/azcore/fake"
+	"github.com/Azure/azure-sdk-for-go/sdk/azcore/to"
```

Comment:
This is not a blocker for PR, but there's "k8s.io/utils/ptr" usages in other places.
We should discuss which we should use.

Reply:
Agreed, I don't see the need for using utils/ptr when there's already azcore/to to do it and we've been using that up until now.